Sources/SkeletonView.swift

```diff
@@ -67,6 +90,36 @@ extension UIView {
         }
     }
     
+    fileprivate func recursiveUpdateSkeleton(withType type: SkeletonType, usingColors colors: [UIColor], animated: Bool, animation: SkeletonLayerAnimation?) {
+        layoutIfNeeded()
+
+        updateDummyDataSourceIfNeeded()
```

Comment:
with this method I think the same, I mean, we need to reload the skeleton but we shouldn't  create a new dummy datasource if it doesn't exist, if so, something was wrong. Do you know want I mean?

Reply:
The same here. The update method acts as a show or update method. So could happen that the user calls the `showOrUpdate` methods before calling any other `show` method. In this case, the `update` method will have to create the `datasource`.

I've updated this method too from `recursiveUpdateSkeleton` to `recursiveShowOrUpdateSkeleton` to reflect the method purpose.